grafana/grafana.ini

```diff
@@ -0,0 +1,23 @@
+[server]
+domain = localhost
+# root_url = %(protocol)s://%(domain)s:%(http_port)s/
+root_url = http://localhost:3000
```

Comment:
@hengsokly pleaes update Grafana to export ports: 8000:3000 and keep rails use the default 3000:3000 as Rails is accessing more often then